docs/data/migration/migration-pickers-v7/migration-pickers-v7.md

```diff
@@ -314,3 +314,111 @@ If you were using them, you need to replace them with the following code:
   > &
     UseDateTimeFieldProps<TDate, TEnableAccessibleFieldDOMStructure>;
   ```
+
+## Stop passing `utils` and the date object to some translation keys
+
+Some translation keys no longer require `utils` and the date object as parameters, but only the formatted value as a string. The keys affected by this changes are: `clockLabelText`, `openDatePickerDialogue` and `openTimePickerDialogue`.
+
+If you have customized those translation keys, you have to update them:
+
+```diff
```

Comment:
Given the increased amount of examples, I no longer strongly prefer grouping them in a single diff. 😉 
If you feel like 3 separate diffs per type of usage make more sense, I'm all for it. 👍